### DISCUSSION TITLE
Resizing to correct monitor-size?

### DISCUSSION THREAD

**Discussion by qns7** [relikd/Memmon#16] [Ideas]
Hey there, 

I came across a tiny problem while gracefully using your memmon: 
If I wake up my macbook memmon realigns my space, so far so good :) But there is one program/app which doesn't fit it's former maximized windows to the external monitor. 
It moves there rightfully, but keeps the size of the inbuilt screen. 
This seems only to be the case with firefox, at least that's the only one I stumbled upon this very issue. 

Perhaps there is a easy fix for this? If not, it doesn't matter really and that behaviour doesn't bother me too much. 
But nevertheless I wanted to tell you this. 

Best regargds Q

**Comment by relikd** [comment]
yes, unfortunately this is known. Firefox and Chrome have issues because they create hidden windows. I have access to these windows only in one direction (read state) but not the other (restore state). See issue #6 and #4. Sadly, I have not found a workaround so far.

I am a bit surprised that it did restore the windows at all. Probably the number of windows was (coincidentally) aligned and Memmon restored to a size of a hidden window. But thats just my guess. I have no idea why there are hidden windows in the first place or how they are used by Firefox.

**Reply by qns7** [reply]
Sorry for not checking the issues before writing this... Have a nice day!
